bot/exts/events/advent_of_code/_cog.py

```diff
@@ -52,13 +60,77 @@ def __init__(self, bot: Bot):
         self.status_task.set_name("AoC Status Countdown")
         self.status_task.add_done_callback(_helpers.background_task_callback)
 
+        self.completer_task.start()
+
+    @tasks.loop(minutes=10.0)
+    async def completer_task(self) -> None:
```

Comment:
For consistency with the role's name, rename this to `completionist_task`.

Reply:
Renamed the task, and the docstrings in here https://github.com/python-discord/sir-lancebot/pull/991/commits/21518aed7b3cfb9c2ec4fa1722689a6c4df56372